### ISSUE TITLE
looks like it has to be checked out in home-folder with a specific key-name

### ISSUE THREAD

**Issue by bpletzer** [volcomism/homestripped#1]
➜  homestead git:(master) vagrant up
/Users/XXX/Developement/homestead/scripts/homestead.rb:94:in `read': No such file or directory - /Users/XXX/.ssh/id_rsa.pub (Errno::ENOENT)

**Comment by volcomism** [comment]
Yes, you will need to modify the 'authorize' and 'key' values in Homestead.yaml to point towards your local ssh-keys.
